<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-16T13:07:44 IST" name="Default suite" finished-at="2023-05-16T13:07:45 IST" duration-ms="1084">
    <groups>
    </groups>
    <test started-at="2023-05-16T13:07:44 IST" name="Default test" finished-at="2023-05-16T13:07:45 IST" duration-ms="1084">
      <class name="com.ssid.test.didCreation.DidCreation">
        <test-method is-config="true" signature="Excel()[pri:0, instance:com.ssid.test.didCreation.DidCreation@3b0090a4]" started-at="2023-05-16T13:07:44 IST" name="Excel" finished-at="2023-05-16T13:07:45 IST" duration-ms="976" status="FAIL">
          <exception class="org.apache.poi.openxml4j.exceptions.InvalidOperationException">
            <message>
              <![CDATA[Can't open the specified file input stream from file: 'C:\Users\colbi\OneDrive\Desktop\Test_Android\Book.xlsx']]>
            </message>
            <full-stacktrace>
              <![CDATA[org.apache.poi.openxml4j.exceptions.InvalidOperationException: Can't open the specified file input stream from file: 'C:\Users\colbi\OneDrive\Desktop\Test_Android\Book.xlsx'
at org.apache.poi.openxml4j.opc.ZipPackage.openZipEntrySourceStream(ZipPackage.java:188)
at org.apache.poi.openxml4j.opc.ZipPackage.<init>(ZipPackage.java:175)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:282)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:191)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:310)
at com.ssid.base.platform.platform.Excel(platform.java:113)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.FileNotFoundException: C:\Users\colbi\OneDrive\Desktop\Test_Android\Book.xlsx (The system cannot find the file specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at org.apache.poi.openxml4j.opc.ZipPackage.openZipEntrySourceStream(ZipPackage.java:185)
... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- org.apache.poi.openxml4j.exceptions.InvalidOperationException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Excel -->
        <test-method signature="didCreation()[pri:0, instance:com.ssid.test.didCreation.DidCreation@3b0090a4]" started-at="2023-05-16T13:07:45 IST" name="didCreation" finished-at="2023-05-16T13:07:45 IST" duration-ms="0" status="SKIP">
          <exception class="org.apache.poi.openxml4j.exceptions.InvalidOperationException">
            <message>
              <![CDATA[Can't open the specified file input stream from file: 'C:\Users\colbi\OneDrive\Desktop\Test_Android\Book.xlsx']]>
            </message>
            <full-stacktrace>
              <![CDATA[org.apache.poi.openxml4j.exceptions.InvalidOperationException: Can't open the specified file input stream from file: 'C:\Users\colbi\OneDrive\Desktop\Test_Android\Book.xlsx'
at org.apache.poi.openxml4j.opc.ZipPackage.openZipEntrySourceStream(ZipPackage.java:188)
at org.apache.poi.openxml4j.opc.ZipPackage.<init>(ZipPackage.java:175)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:282)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:191)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:310)
at com.ssid.base.platform.platform.Excel(platform.java:113)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.FileNotFoundException: C:\Users\colbi\OneDrive\Desktop\Test_Android\Book.xlsx (The system cannot find the file specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at org.apache.poi.openxml4j.opc.ZipPackage.openZipEntrySourceStream(ZipPackage.java:185)
... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- org.apache.poi.openxml4j.exceptions.InvalidOperationException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- didCreation -->
      </class> <!-- com.ssid.test.didCreation.DidCreation -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
